lib/api/configs.js

```diff
@@ -0,0 +1,80 @@
+var linz = require('../../'),
+    async = require('async'),
+    clone = require('clone');
+
+/**
+ * Get linz config by reference or by cloning a copy
+ * @param  {String} configName      Name of config
+ * @param  {Boolean} copy           Specify if result should be a cloned copy or by reference. Default to return a reference.
+ * @return {Object}                 Config object either by a cloned copy or by reference
+ */
+function get (configName, copy) {
+
+    if (copy) {
+        return clone(linz.get('configs')[configName]);
+    }
+
+    return linz.get('configs')[configName];
+}
+
+
+/**
+ * Get config permissions
+ * @param {Object}   req      Request data
+ * @param {String}   configName Config name
+ * @param {Function} cb        Callback function
+ * @return {Object}
+ */
+function getPermissions(req, configName, cb) {
+
+    if (!linz.get('disabled config cache') && req.session.configs) {
+        return cb(null, req.session.configs);
+    }
+
+    var config = get(configName);
```

Comment:
There doesn't seem to be any support here for skipping this code if the default function hasn't been replaced? Did you not want to include that? It's just, if people don't replace the default config, this code will run unnecessarily as we know everything defaults to true.